libs/application/templates/parental-leave/src/lib/directorateOfLabour.utils.spec.ts

```diff
@@ -0,0 +1,353 @@
+    employers: [{ email: null, nationalRegistryId: 'nationalRegistryId' }],
+    status: 'In Process',
+    rightsCode: null,
+  }
+
+  const parentalLeaves = [
+    {
+      ...mockUser,
+      expectedDateOfBirth: '2016-10-19',
+      dateOfBirth: '2016-10-18',
+      periods: [
+        {
+          from: '2016-10-01',
+          to: '2016-10-31',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2016-11-01',
+          to: '2016-11-30',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2016-12-01',
+          to: '2016-12-31',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2017-03-01',
+          to: '2017-03-31',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2017-04-01',
+          to: '2017-04-30',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+      ],
+    },
+    {
+      ...mockUser,
+      expectedDateOfBirth: '2017-10-02',
+      dateOfBirth: '2017-10-06',
+      periods: [
+        {
+          from: '2017-10-01',
+          to: '2017-10-31',
+          ratio: 100,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2017-11-01',
+          to: '2017-11-30',
+          ratio: 100,
+          approved: true,
+          paid: false,
+        },
+        {
+          from: '2017-12-01',
+          to: '2017-12-31',
+          ratio: 100,
+          approved: true,
+          paid: false,
+        },
+        {
+          from: '2018-01-01',
+          to: '2018-01-31',
+          ratio: 100,
+          approved: true,
+          paid: false,
+        },
+        {
+          from: '2018-02-01',
+          to: '2018-02-28',
+          ratio: 100,
+          approved: true,
+          paid: false,
+        },
+        {
+          from: '2018-03-01',
+          to: '2018-03-31',
+          ratio: 100,
+          approved: true,
+          paid: false,
+        },
+      ],
+    },
+    {
+      ...mockUser,
+      expectedDateOfBirth: '2019-02-03',
+      dateOfBirth: '2019-02-06',
+      periods: [
+        {
+          from: '2019-05-01',
+          to: '2019-05-31',
+          ratio: 50,
+          approved: true,
+          paid: true,
+        },
+        {
+          from: '2019-06-01',
+          to: '2019-06-30',
+          ratio: 50,
+          approved: true,
+          paid: false,
+        },
+        {
+          from: '2019-07-01',
+          to: '2019-07-31',
+          ratio: 50,
+          approved: true,
+          paid: false,
+        },
+      ],
+    },
+  ]
+
+  const availableRights = { independentMonths: 6, transferableMonths: 1.5 }
+  const unavailableRights = { independentMonths: 0, transferableMonths: 0 }
+  const firstDob = '2016-10-18'
+  const secondDob = '2017-10-06'
```

Comment:
When number of months since dob >= 24 you lose rights to take parental leave. As always there are some edge cases, but we won't cover them in this phase.

Reply:
The date of births here are not used together, it's just different test cases with different dates